Update a2_questionnaire_design_part_a.md

ShrutiPatil-SAP opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Answered the questions.
## What did you learn from the changes you have made?

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

## How were these changes tested?

## A reference to a related issue in your repository (if applicable)

## Checklist
- [ ] I can confirm that my changes are working as intended

ShrutiPatil-SAP committed:
Update a2_questionnaire_design_part_a.md

amanda-ng518 reviewed (changes requested):
A2 observational units missing

ShrutiPatil-SAP committed:
Update a2_questionnaire_design_part_a.md by adding Observational units

ShrutiPatil-SAP commented:
Hi Amanda, thanks for pointing it out. I have updated the assignment and
added the observational units. Should I create a new pull request?

Regards,
Shruti.

On Tue, Nov 12, 2024 at 11:04 AM amanda-ng518 ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> A2 observational units missing
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ShrutiPatil-SAP/sampling/pull/2#pullrequestreview-2430036486>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKQP5TVDHQUSRCSHYPQSHPT2AIRKHAVCNFSM6AAAAABRQSLRISVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDIMZQGAZTMNBYGY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

amanda-ng518 reviewed (approved):
A2 pass
